README.md

```diff
@@ -55,17 +55,17 @@ ssh, scp, rsync:
 ## Environment
 
 - Ubuntu 14.04
-- Drupal 7.43
-- MySQL 5.5.47
+- Drupal 7.50
+- MySQL 5.5.49
```

Comment:
@DonRichards @ruebot 

I just brought up a fresh "islandora_vagrant_base_box" box it is showing 7.50?

root@islandora:/var/www/drupal# drush status
 Drupal version                  :  7.50
 Site URI                        :  http://default
 Database driver                 :  mysql
 Database username               :  root
 Database name                   :  drupal7
 Database                        :  Connected
 Drupal bootstrap                :  Successful
 Drupal user                     :  Anonymous
 Default theme                   :  bartik
 Administration theme            :  seven
 PHP executable                  :  /usr/bin/php
 PHP configuration               :  /etc/php5/cli/php.ini
 PHP OS                          :  Linux
 Drush version                   :  6.3.0
 Drush configuration             :
 Drush alias files               :
 Drupal root                     :  /var/www/drupal
 Site path                       :  sites/default
 File directory path             :  sites/default/files
 Temporary file directory path   :  /tmp